### ISSUE TITLE
Feature Request: Add changelog to release notes.

### ISSUE THREAD

**Issue by seawatts** [html-next/ember-cli-changelog#11]
I feel like there has to be some coordination between ember-cli-release and this changelog. It would be cool if ember-cli-release could actually create a github release. And then this could add the changelog to the release notes. Similar to how ember-cli does their releases. 

Thoughts?

**Comment by runspired** [comment]
@seawatts this is already built in, but a tighter integration is coming. The default blueprint should have installed the hook into your release config.

**Comment by seawatts** [comment]
@runspired do you mean that it will annotate a tag and then if you go to github and click draft new release, the changelog will be prepopulated by the selected tag.

**Comment by runspired** [comment]
@seawatts when you `ember release`, it uses the `beforeCommit` hook to ask you if you want to generate the changelog for the release, then does so, then prompts you to make any final alterations, then adds it to the commit files for the release.

**Comment by runspired** [comment]
Realizing we may be talking about different things here.

**Comment by ryanjm** [comment]
@seawatts Are you talking about this type of [github release](https://help.github.com/articles/creating-releases/)?

If so, it seems like that would be it's own addon since it would have to deal with github's API. That said, if that addon could use an `ember-cli-changelog` hook to get the full log entry for a given release, that'd help make that an easier addon, right?

**Comment by seawatts** [comment]
Yes I am. But this is already using the github api https://github.com/runspired/ember-cli-changelog/blob/21de29fda5f9610b90b15c88b4a11b5f528f60a6/lib/helpers/git/github/compare-commits.js#L4
